src/main/java/bank/account/AccountDTO.java

```diff
@@ -0,0 +1,11 @@
+package bank.account;
+
+import bank.person.PersonDTO;
+
+public record AccountDTO(
+        String id,
+        String iban,
+        int balance,
+        PersonDTO person
```

Comment:
власне тут я б давав лише 
```suggestion
        String personId
```
якщо у тебе ресурс "рахунок (аккаунт)" то навіщо тобі повний ресурс персони? а ось ідентифікатор персони - в самий раз.